### ISSUE TITLE
Environment.yml use for a Mac

### ISSUE THREAD

**Issue by jomey** [snowex-hackweek/docker-image#27]
Not an issue for the Docker and Linux environments and when people try to re-use the `environment.yml` to replicate it on a Mac, they will fail with this entry:
```yml
python-pdal=2.4=py38h1fd1430_1
```
The hash at the end seems to be version specific to the Linux version.

**Comment by scottyhq** [comment]
right. PDAL was a pain to install, had to use that specific version for the import to work (https://github.com/snowex-hackweek/docker-image/pull/12). I think running via docker is the only surefire way to get the same environment, but you can also try using https://github.com/conda-incubator/conda-lock to export lock files for each OS... probably do need to drop any specific build hashes though.

**Comment by jomey** [comment]
You can loosen the hash restriction and just pin it down to that version
```
python-pdal=2.4
```

That will work for all OSes and only get that published one. The hash will pin it down to flavor and version of OS.

**Comment by jomey** [comment]
From https://anaconda.org/conda-forge/python-pdal/files

For completeness, this would be the corresponding Mac version:
```
 osx-64/python-pdal-2.4.0-py38h12bbefe_1
```
